src/collection-proxy.js

```diff
@@ -416,6 +416,16 @@ class CollectionProxy {
     return collection
   }
 
+  /**
+  * Returnes reversed version of original collection

```

Comment:
Can you please fix the typo in `Returnes` ?

```suggestion
  * Returns reversed version of the original collection
```